### PR TITLE
Benchmarking Snowflake arctic-instruct feedback function of groundedness

### PR DESCRIPTION
**Pull request by sfc-gh-dhuang**
Items to add to release announcement:
- **Heading**: Benchmarking Snowflake arctic-instruct feedback function of groundedness against 500 entries of SummEval test set

Other details that are good to know but need not be announced:
highlighting the enterprise intelligence (efficiency + good alignment with human golden set) on groundedness eval
image

dataset ref: https://huggingface.co/datasets/mteb/summeval

### CODE REVIEW

File: trulens_eval/trulens_eval/tests/groundedness_benchmark_snowflake_arctic.ipynb [hunk 1]
```diff
@@ -0,0 +1,400 @@
+{
```
Comment (joshreini1) [inline trulens_eval/trulens_eval/tests/groundedness_benchmark_snowflake_arctic.ipynb:1]: <p>if easy, can you show turbo in the same leaderboard as the others?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/1185/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (sfc-gh-jreini) [reply trulens_eval/trulens_eval/tests/groundedness_benchmark_snowflake_arctic.ipynb:1]: Would be great to keep the NLI results here as well

Comment (sfc-gh-pdharmana) [reply trulens_eval/trulens_eval/tests/groundedness_benchmark_snowflake_arctic.ipynb:1]: <p>+1 if the command is same <code>get_leaderboard</code>, why does it get two different outputs ?</p>

Comment (sfc-gh-dhuang) [reply trulens_eval/trulens_eval/tests/groundedness_benchmark_snowflake_arctic.ipynb:1]: @sfc-gh-pdharmana there's a bug at the moment that Piotr is working on which prevents us to initialize TruBasicApp in a row. 
Basically defining more than one TruBasicApp in a row and calling recording with context does not work - only the recorder defined last in the order will trigger adding new records to DB. Refer to this [notebook](https://github.com/truera/trulens/blob/7c89f53a7fb15836e1391dfebe804c4f32011bec/trulens_eval/examples/quickstart/text2text_quickstart.ipynb) for a minimal reproducible example

**Comment by sfc-gh-dhuang** [comment]
Adding results for Mixtral-8x7b (mixture of expert) and Llama3-70B here as well 
image

File: trulens_eval/examples/quickstart/text2text_quickstart.ipynb [hunk 2]
```diff
@@ -141,22 +151,70 @@
...
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 20,
    "metadata": {},
    "outputs": [],
    "source": [
     "from trulens_eval import TruBasicApp\n",
-    "tru_llm_standalone_recorder = TruBasicApp(llm_standalone, app_id=\"Happy Bot\", feedbacks=[f_answer_relevance])"
+    "\n",
+    "# Daniel - notice a regression behavior here, where only the recorder defined last (tru_llm_standalone_recorder_2) will be recording, anything above does not perform recording at all.\n",
```
Comment (sfc-gh-jreini) [inline trulens_eval/examples/quickstart/text2text_quickstart.ipynb:160]: is this change intended? or just a temporary marker of a bug

Comment (sfc-gh-dhuang) [reply trulens_eval/examples/quickstart/text2text_quickstart.ipynb:160]: oops this was the minimal example I set up to repro the bug of TruBasicApp. Piotr is aware of this and I'll remove this change

**Review by sfc-gh-jreini — APPROVED** [review]
LGTM - I've added these numbers to: https://github.com/truera/trulens/pull/1190
